package.json

```diff
@@ -0,0 +1,137 @@
+    "lint:sample:fix": "eslint sample/**/*.{ts,tsx} --quiet --fix",
+    "prettier:sample": "prettier --check sample/**/*.ts*",
+    "prettier:sample:fix": "prettier --write sample/**/*.ts*",
+    "codestyle:fix": "npm run lint:fix && npm run prettier:fix",
+    "codestyle:sample:fix": "npm run lint:sample:fix && npm run prettier:sample:fix",
+    "codestyle": "npm run codestyle:fix && npm run codestyle:sample:fix"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unosquare/uno-react-draftjs"
+  },
+  "keywords": [
+    "sharepoint",
+    "webpart",
+    "oneonone"
+  ],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/unosquare/uno-react-draftjs/issues"
+  },
+  "homepage": "https://github.com/unosquare/uno-react-draftjs#readme",
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@testing-library/jest-dom": "^5.11.6",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.5.0",
+    "@types/jest": "^26.0.16",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "17.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.9.1",
+    "@typescript-eslint/parser": "^4.9.1",
+    "babel-loader": "^8.2.2",
+    "eslint": "^7.15.0",
+    "eslint-config-prettier": "^7.0.0",
+    "eslint-plugin-prettier": "^3.2.0",
+    "eslint-plugin-react": "^7.21.5",
+    "html-webpack-plugin": "^4.5.0",
+    "jest": "^26.6.3",
+    "jest-fetch-mock": "^3.0.3",
+    "prettier": "^2.2.1",
+    "react-is": "^17.0.1",
+    "source-map-loader": "^1.1.3",
+    "ts-jest": "^26.4.4",
+    "ts-loader": "^8.0.11",
+    "typescript": "^4.1.2",
+    "webpack": "^5.10.0",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.0",
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
+    "react-select": "3.1.0"
+  },
+  "peerDependencies": {
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
+    "react-select": "3.1.0"
+  },
+  "dependencies": {
+    "@fluentui/react": "^7.153.4",
+    "@microsoft/applicationinsights-react-js": "^3.0.5",
+    "@microsoft/applicationinsights-web": "^2.5.10",
+    "@progress/kendo-drawing": "^1.9.3",
+    "@types/draft-js": "^0.10.44",
+    "draft-js": "^0.11.7",
+    "history": "^5.0.0",
+    "libphonenumber-js": "^1.9.6",
+    "markdown-draft-js": "^2.2.1",
+    "tubular-common": "^5.0.1",
+    "tubular-fabric": "^3.0.1",
+    "tubular-react-common": "^3.0.1",
+    "uno-react": "^1.0.16"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js"
+    ],
+    "transform": {
+      "^.+\\.(ts|tsx)$": "ts-jest"
+    },
+    "testMatch": [
+      "**/src/**/*.test.+(ts|tsx|js)"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 50,
+        "functions": 50,
+        "lines": 80,
+        "statements": -230
+      }
+    },
+    "moduleNameMapper": {
```

Comment:
These mappers are for Fluent, if this component is not fluent, remove it.

Reply:
The component uses fluent styles.

Reply:
Aright then.